packages/shared/src/components/onboarding/OnboardingFeedHeader.tsx

```diff
@@ -95,21 +96,21 @@ export const OnboardingFeedHeader = ({
 
   return (
     <LayoutHeader className="flex-col overflow-x-visible">
-      <div className="fixed z-1 flex w-full justify-center bg-gradient-to-b from-overlay-primary-pepper from-90% to-transparent py-10">
-        <div className="flex w-[40rem] rounded-16 border-l-2 border-l-cabbage-10 bg-theme-bg-notification p-6">
-          <div className="mr-auto">
+      <div className="fixed z-1 flex w-full justify-center bg-gradient-to-b from-theme-surface-invert via-80% tablet:px-4 tablet:py-10">
+        <div className="flex h-44 w-full max-w-[40rem] flex-col items-center border-l-2 border-theme-color-cabbage bg-theme-bg-popover  p-6 tablet:h-auto tablet:flex-row tablet:rounded-16">
+          <div className="text-center tablet:text-left">
             <p className="font-bold typo-title3">
               Pick tags that are relevant to you
             </p>
             <p className="text-theme-label-tertiary typo-callout">
               You can always modify your tags later
             </p>
           </div>
-
+          <div className="flex flex-1" />
```

Comment:
Decided to leverage a fixed height on mobile, it was easier to maintain the padding on the element below to always match.